package.json

```diff
@@ -62,8 +62,8 @@
     "node-mocks-http": "^1.6.6"
   },
   "peerDependencies": {
-    "fusion-core": ">=0.3.0-2",
-    "fusion-tokens": "^0.0.4"
+    "fusion-core": "0.3.0-5",
```

Comment:
Should this be a loose dependency range?

Reply:
I worry issues like https://github.com/fusionjs/fusion-core/issues/95 will turn into a breaking change, so I'm going to leave it pinned and see how it goes